cloudferrylib/os/storage/cinder_netapp.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from cloudferrylib.os.storage import cinder_database
+from cloudferrylib.utils import utils
+
+
+LOG = utils.get_log(__name__)
+
+
+class CinderNetApp(cinder_database.CinderStorage):
+
+    """Migration strategy used with NetApp multi backend.
+
+    Extends `cinder_database` functionality with defining which volumes
+    are stored on each particular NetApp server.
+
+    To use this strategy - specify cinder_migration_strategy in config as
+    'cloudferrylib.os.storage.cinder_netapp_CinderNetApp'.
```

Comment:
`'cloudferrylib.os.storage.cinder_netapp_CinderNetApp'` has underscore instead of period before `CinderNetApp`